lib/osc.py

```diff
@@ -285,17 +285,13 @@ def _wait_for_job(self, data, headers="JSON"):
     # Get the ISC version.
     # Return a string with the ISC version
     def getISCVersion(self):
-        try:
-            self._output.log_info("inside getISCVersion")
-            action = 'ISC version query'
-            url = '/api/server/v1/serverManagement/status'
-            body = ''
+        action = 'ISC version query'
+        url = '/api/server/v1/serverManagement/status'
```

Comment:
Please don't change the getISCVersion method
It will revert my changes and make automation fail.

Reply:
All changes done by rebase, edit, add, commit, fetch.